charts/datadog-operator/templates/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "datadog-operator.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
the file is part of the `datadog-operator` chart not the `datadog` chart. 
the change still make sense, but it require to also:
* bump the `datadog-operator` chart version
* update the `datadog-operator` changelog.md file.

Reply:
updated, ty!